llrpConnector/README.md

```diff
@@ -0,0 +1,238 @@
+empty.
+*  **hostame** - Required. The hostame or IP Address of the RFID Reader
+*  **readerPort** - Required. The readerPort to connect to the RFID Reader, which the LLRP protocol specification specifies port 5084 is used for data transfer.
+*  **tagReadInterval** - Optional. Interval in milliseconds to receive collected Tag Data from the RFID Reader. (Default: 500)
+*  **logLevel** - Optional. Indicates log level for messages to be sent to Vantiq ('info', 'warn', 'error', 'debug' are allowed). (Default: send no log messages)
+
+## Select and Publish Statements
+
+There is no support for either Select or Publish statements from Vantiq.  Once a connection is successfully made, messages will automatically be sent to Vantiq based on the Vantiq Source configuration properties.
+
+## Messages Sent To Vantiq Source
+
+### Log Messages
+
+Receipt of any log messages sent from the LLRP Connector will be based on the `logLevel` property in the [Vantiq Source Configuration](#sourceConfig).  This property is optional and if not specified no messages will be sent to the Vantiq Source. The following log level values are supported:
+* **error** - Unexpected error conditions
+* **warn** - Disconnection information
+* **info** - Successful connection information
+* **debug** - General messages sent/received to/from the RFID Reader
+
+The `loglevel` is hierarchical in that specififying a lower level will send all log levels up to the desired level.  For example, specifying `info` will send `error`, `warn`, and `info` log messages.
+
+The JSON format of a Log message contains the following properties:
+* **eventType** - Either `errorLog`, `warnLog`, `infoLog`, or `debugLog`, depending on the specified `logLevel` in the Vantiq source configuration.
+* **sourceName** - The Vantiq source name which is used to differentiate when multiple RFID Readers are connected to the same LLRP Connector.
+* **hostname** - The RFID Reader's hostname 
+* **readerId** - The RFID Reader's ID
+* **msg** - Log message provided
+
+Log Message Example:
+```
+{   "eventType": "infoLog",
+    "sourceName": "ZebraRFID",
+    "hostname": "fx7500fcc3e9",
+    "readerId": "84248dfffffcc3e9",
+    "msg": "Connection to Reader was successful"
+}
+```
+**Note**: To capture the Reader Capabilities and Reader Configuration, set the `logLevel` to `debug`.  This information can be used when using a new RFID Reader and want to understand the capabilities of the RFID Reader and configuration settings it was set up to use.
+
+### Reader Status Message
+
+Whenever the connection between the LLRP Connector and the RFID Reader changes, a Reader Status message is sent to the Vantiq Source.
+
+The JSON format of Reader Status message contains the following properties:
+* **eventType** - `readerStatus`
+* **readerId** - The RFID Reader's ID
+* **readerOnline** - Boolean value where `True` is `Online` and 'False` is `Offline`
+
+Log Message Example:

```

Comment:
I'll use a different example, this is slightly different because you can connect to the reader, but there is a set of messages between the reader and the connector that needs to happen before it is classified as online.